src/components/ec-summary-info/ec-summary-info.story.js

```diff
@@ -0,0 +1,76 @@
+import EcIcon from '../ec-icon';
+import EcSummaryInfo from './ec-summary-info.vue';
+
+export default {
+  title: 'Summary Info',
+  component: EcSummaryInfo,
+};
+
+const Template = args => ({
+  components: { EcSummaryInfo },
+  setup() {
+    return {
+      args,
+    };
+  },
+  template: `
+    <ec-summary-info
+      v-bind="args"
+    />
+  `,
+});
+
+export const basic = Template.bind({});
+basic.args = {
+  title: 'This is the title',
+  iconName: 'simple-sell',
+};
+
+export const withContentLinesProp = args => ({
+  components: { EcSummaryInfo },
+  setup() {
+    return { args };
+  },
+  template: `
+    <ec-summary-info
+      v-bind="args"
+    />
+  `,
+});
+
+withContentLinesProp.args = {
+  ...basic.args,
+  lineItems: [
+    {
+      text: 'Some text',
+      cssClasses: ['tw-text-gray-4', 'tw-uppercase'],
+      iconCssClasses: ['tw-h-16'],
```

Comment:
For now I'd not create this cssClasses options in line Items and stick with preset for line only(style property in lineItems in the task). We could name it preset  instead of style to not confuse it with style attribute.

Reply:
ok, corrected. thanks